x-pack/plugins/cases/public/components/connectors/servicenow/servicenow_itsm_case_fields.tsx

```diff
@@ -149,90 +152,111 @@ const ServiceNowITSMFieldsComponent: React.FunctionComponent<
     }
   }, [category, impact, onChange, severity, subcategory, urgency]);
 
-  return isEdit ? (
-    <div data-test-subj={'connector-fields-sn-itsm'}>
-      <EuiFormRow fullWidth label={i18n.URGENCY}>
-        <EuiSelect
-          fullWidth
-          data-test-subj="urgencySelect"
-          options={urgencyOptions}
-          value={urgency ?? undefined}
-          isLoading={isLoadingChoices}
-          disabled={isLoadingChoices}
-          hasNoInitialSelection
-          onChange={(e) => onChangeCb('urgency', e.target.value)}
-        />
-      </EuiFormRow>
-      <EuiSpacer size="m" />
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiFormRow fullWidth label={i18n.SEVERITY}>
-            <EuiSelect
-              fullWidth
-              data-test-subj="severitySelect"
-              options={severityOptions}
-              value={severity ?? undefined}
-              isLoading={isLoadingChoices}
-              disabled={isLoadingChoices}
-              hasNoInitialSelection
-              onChange={(e) => onChangeCb('severity', e.target.value)}
+  return (
+    <>
+      {showConnectorWarning && (
```

Comment:
The `DeprecatedCallout` has been added. The other code has been wrapped with `<>`